### DISCUSSION TITLE
Instead of uploading images, also support providing direct image link

### DISCUSSION THREAD

**Discussion by kausar007** [benjaminjonard/koillection#1098] [Ideas]
For example when we choose image in the item, then instead of asking to upload file. There should be 2 options, either upload file or paste a direct link to image such as https://upload.wikimedia.org/wikipedia/commons/thumb/d/de/Solar_eclipse_of_April_2024_from_Indianapolis.jpg/480px-Solar_eclipse_of_April_2024_from_Indianapolis.jpg

This will reduce storage utilization. The UI should load image only when item opened.

**Comment by risalt** [comment]
Hi, This would be useful for me too and should not be too difficult to implement: In my use case, I just populate the database using an external tool instead of the web interface, and I overwrite the database "image"/"video" fields with a path relative to the koillection installation (for instance: "/uploads/pictures/external/collection1/item%201.png", with the typical percent encoding), and it works.
Robert.
